target/linux/ipq40xx/files/arch/arm/boot/dts/qcom-ipq4018-rt-ac58u.dts

```diff
@@ -289,6 +277,34 @@
 	status = "okay";
 };
 
+&gmac {
+	status = "okay";
+};
+
+&switch {
+	status = "okay";
+};
+
+&swport1 {
```

Comment:
I think this needs an additional label, it should be the same as zyxel,nbg6617

Reply:
I dont really know, it was just picked from a contributed commit.